lib/Firtool/Firtool.cpp

```diff
@@ -710,6 +718,7 @@ circt::firtool::FirtoolOptions::FirtoolOptions()
   chiselInterfaceOutDirectory = clOptions->chiselInterfaceOutDirectory;
   vbToBV = clOptions->vbToBV;
   noDedup = clOptions->noDedup;
```

Comment:
I guess we explicitly initialize all the fields on the constructor (above), maybe should do that for this as well?